aspnetcore/fundamentals/host/hosted-services/samples/3.x/BackgroundTasksSample/Services/QueuedHostedService.cs

```diff
@@ -32,25 +30,22 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
 
             _backgroundTask = Task.Run(async () =>
```

Comment:
#7640 removed the Task.Run because it has been extracted into Backgroundservice. Why reintroduce it?